### DISCUSSION TITLE
0.5.68-preview

### DISCUSSION THREAD

**Discussion by gfraiteur** [metalama/Metalama#60] [Changelog]
0.5.68-preview was released:
* Deploying the previously undeployed fixes on [try.metalama.net](http://try.metalama.net/),
* Fixed: In compiler-visible properties, semicolon cannot be used as a separator because it marks a comment. (Metalama-defined ItemGroups did not work)
* The documentation to migrate PostSharp to Metalama has also been released today: https://doc.metalama.net/migration/migration. 
* The Metamala.Migration package now has [Obsolete] annotations everywhere with indications about how to migrate.

**Comment by YBAZAN** [comment]
Thank you for providing release notes for incremental releases!
Much appreciated!
